src/explanations/usdt.yaml

```diff
@@ -0,0 +1,65 @@
+
+
+    ### How They are Backed
+
+
+    USDT is meant to be backed 1:1 by the stable asset, meaning that for every USDT in circulation, Tether Limited is supposed to hold an equivalent value of the stable asset in reserve. According to Tether Limited, the stable asset backing USDT is typically held in the form of US Treasury bills, which are low-risk, short-term debt securities issued by the US government. However, as mentioned above, the company has not undergone a full audit to prove that it has the necessary reserves to back all of the USDT in circulation.
+
+
+    ### Issues and Hacks
+
+
+    Tether Limited and its affiliated exchange, Bitfinex, have faced several controversies over the years. One of the main controversies is the allegation of price manipulation. In April 2017, the US Commodity Futures Trading Commission (CFTC) issued subpoenas to Tether Limited and Bitfinex as part of an investigation into potential market manipulation. In 2018, researchers Griffin and Shams published a study suggesting that Bitcoin prices increased after Tether minted new USDT during market downturns, potentially as an attempt at market manipulation. However, this conclusion was disputed by Bitfinex, the cryptocurrency exchange that is affiliated with Tether, and subsequent researchers have found little to no evidence that Tether's issuance of USDT influenced Bitcoin prices.
+
+
+    Another controversy surrounding Tether is its lack of transparency and audits. Tether claims that its USDT stablecoin is backed 1:1 by US dollars held in reserve, but the company has not undergone a full audit to prove this. In 2018, the New York Attorney General's office filed a lawsuit against Tether and Bitfinex, alleging that the companies had engaged in a cover-up to hide the loss of $850 million in client and corporate funds. The case is ongoing.
+
+
+    Tether and Bitfinex have also faced security and liquidity issues. In August 2016, Bitfinex was hacked and lost approximately 120,000 bitcoins, worth over $70 million at the time. In November 2017, $31 million worth of USDT was stolen from the company. Tether has also struggled to meet withdrawal requests in the past and has not produced an audit showing that it has the necessary reserves to back all of the USDT in circulation.
+
+
+    In September 2022, Bitfinex and Tether (referred to in court records as B/T) were ordered by the New York District Court to produce documents showing the backing of USDT, the outcome of which is still pending. The court also ordered B/T to stop loaning out Tether's stablecoins and to provide a full accounting of the Tether reserve. You can read more about it [here](https://cointelegraph.com/news/new-york-judge-orders-tether-to-document-usdt-backing).
+
+
```

Comment:
We should add another section, "growth and adoption", which should be on the positive side as USDT is the largest stable coin.